src/components/Transaction/PaginationList/index.tsx

```diff
@@ -1,34 +1,38 @@
 import React, { ReactNode, useState } from 'react'
 import LoadMoreIcon from '../../../assets/transaction_load_more.png'
-import { InputOutput } from '../../../http/response/Transaction'
-import TransactionCell from '../Cell'
 import Container from './styled'
 
 interface Props {
```

Comment:
Please use more meaningful name for this interface name

Reply:
https://github.com/nervosnetwork/ckb-explorer-frontend/pull/51/commits/ae55b4655737c0533220c535dc60a0fc2055280b